packages/typography/src/Paragraph/Paragraph.tsx

```diff
@@ -53,18 +53,16 @@ export const Paragraph = ({
   classNameOverride,
   ...restProps
 }: ParagraphProps): JSX.Element => {
-  const className = classnames([
+  const className = classnames(
     styles.paragraph,
     styles[variant],
     styles[color],
-    classNameOverride,
-  ])
+    classNameOverride
+  )
 
   return createElement(
     tag === undefined ? "p" : tag,
     { ...restProps, className },
     children
   )
 }
-
-Paragraph.displayName = "Paragraph"
```

Comment:
Did you mean to remove `displayName` from some components in this PR @HeartSquared?